Notebooks/OOPS_TMT_Statistical_testing.Rmd

```diff
@@ -98,19 +106,23 @@ run_limma <- function(data_to_model,
 }
 ```
 
-build the volcano_plot function
+Build the volcano_plot function
 
 ```{r}
+
+
 volcano_plot <- function(lmResultsObj,
```

Comment:
Updated the `volcano_plot` function so it can handle results tables from `limma` and `DEqMS`. Just needs arguments for the column with p-values and adjusted p-values